trajopt/src/collision_terms.cpp

```diff
@@ -917,7 +917,8 @@ SingleTimestepCollisionEvaluator::SingleTimestepCollisionEvaluator(
 
   contact_manager_ = env_->getDiscreteContactManager();
   contact_manager_->setActiveCollisionObjects(adjacency_map_->getActiveLinkNames());
-  contact_manager_->setContactDistanceThreshold(safety_margin_data_->getMaxSafetyMargin() + safety_margin_buffer_);
+  /** @todo Should remove trajopt safety margin data structure and use the one from tesseract */
```

Comment:
I believe I did this in #219 for trajopt_ifopt.

Reply:
Great

> Thanks for doing this.

No problem.